stacks-101/proof-of-transfer.md

```diff
@@ -60,7 +60,7 @@ We also recommend [reading the full PoX whitepaper](https://community.stacks.org
 
 ### Proof of Transfer Contracts and Technical Details
 
-The Proof of Transfer functionality is implemented on the Stacks chain via a [Clarity smart contract](https://explorer.stacks.co/txid/0xfc878ab9c29f3d822a96ee73898000579bdf69619a174e748672eabfc7cfc589). An overview of this contract is available in the docs.
+The Proof of Transfer functionality is implemented on the Stacks chain via a [Clarity smart contract](https://explorer.hiro.so/txid/SP000000000000000000002Q6VF78.pox?chain=mainnet). An overview of this contract is available in the docs.
```

Comment:
please use the current contract: https://explorer.hiro.so/txid/SP000000000000000000002Q6VF78.pox-4?chain=mainnet

Reply:
done